src/components/AppSidebar/AppSidebar.vue

```diff
@@ -725,14 +733,6 @@ $top-buttons-spacing: 6px;
 				margin-top: -2px;
 			}
 
-			.app-sidebar-header__tertiary-actions {
```

Comment:
This needs to stay, see my comment https://github.com/nextcloud/nextcloud-vue/pull/2715#issuecomment-1145352166